### PR TITLE
MGMT-2659 DHCP server changes the VIPs addresses after installation has started

### PR DESCRIPTION
**Pull request by ori-amizur**
- Return VIPs leases as part of the DHCP allocation response
- Set lease files if provided by DHCP allocation request
- UT and subsystem

/cc @YuviGold 
/cc @ronniel1

### CODE REVIEW

File: subsystem/lease_test.go [hunk 1]
```diff
@@ -153,8 +168,109 @@ var _ = Describe("Lease tests", func() {
...
+func getTcpdumpReponse(hostID string) string {
+	EventuallyWithOffset(1, func() bool {
+		return isReplyFound(hostID, &TcpdumVerifier{})
+	}, 30*time.Second, 5*time.Second).Should(BeTrue())
+
+	stepReply := getSpecificStep(hostID, &TcpdumVerifier{})
+	return stepReply.Output
+}
+
+func countIndexes(s, substr string) int {
```
Comment (YuviGold) [inline subsystem/lease_test.go:256]: What's the meaning of this function?

File: src/dhcp_lease_allocate/main_test.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func newDependenciesMock() *MockDependencies {
+	return &MockDependencies{}
+}
+
+func TestSubsystem(t *testing.T) {
```
Comment (YuviGold) [inline src/dhcp_lease_allocate/main_test.go:14]: This isn't subsystem

File: subsystem/lease_test.go (hunk 1 above)
Comment (ori-amizur) [reply subsystem/lease_test.go:256]: Changed the name.  Actually it is countSubstringOccurances

File: src/dhcp_lease_allocate/main_test.go (hunk 2 above)
Comment (ori-amizur) [reply src/dhcp_lease_allocate/main_test.go:14]: Changed the name

**Review by YuviGold — APPROVED** [review]
LGTM

**Review by YuviGold — APPROVED** [review]

**Pull request merged**